assignments/array-methods.js

```diff
@@ -55,29 +55,72 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
-
+let fullNames = [];
+runners.forEach(getFullName);
+function getFullName(runner){
+    let fullName = runner.first_name + " " + runner.last_name;
+    fullNames.push(fullName);
+}
+console.log(fullNames);
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(function(runner) {
+    let upName = runner.first_name.toUpperCase();
+    allCaps.push(upName);
```

Comment:
You don't need to create a new array. you can do something like 

```javascript 
let allCapsAgain = runners.map(function(person) {
	let name = person.first_name.toUpperCase();
        return name
});
console.log(allCapsAgain)
```


`The map() method creates a new array with the results of calling a function for every array element.

The map() method calls the provided function once for each element in an array, in order.`